### ISSUE TITLE
Bug in ZoomableContainer

### ISSUE THREAD

**Issue by NikolasAst** [amcharts/amcharts5#1326]
The bug is a bit hard to describe, I will try with an example below.

Let's say that I position the graph to the bottom right corner of the container ( example on image below) 

<img width="460" alt="image" src="https://github.com/amcharts/amcharts5/assets/75013148/e47168b3-9cc9-4538-993a-22fac13a9d59">

 When trying to drag the nodes to the upper left corner of the container, the logic breaks and the nodes get hidden behind the canvas border.

<img width="462" alt="image" src="https://github.com/amcharts/amcharts5/assets/75013148/42737aea-b004-48d9-9758-dd7d56e0de9f">

This bug is even more noticeable on zoom-out ( using the minZoomLevel option ) and when panning a graph that the nodes overfill outside the borders.

**Comment by zeroin** [comment]
Indeed, thanks. You can solve this by adding  `maskContent:false` for the series. I modified the example.

**Comment by NikolasAst** [comment]
Ok, that does it. Thanks

**Comment by choconutzy** [comment]
> Indeed, thanks. You can solve this by adding `maskContent:false` for the series. I modified the example.

can we put scrollbar in this zoomable container for unlimited nodes?
